engine/include/cubos/engine/gizmos/gizmos.hpp

```diff
@@ -0,0 +1,89 @@
+/// @file
+/// @brief Resource @ref cubos::engine::Gizmos.
+/// @ingroup gizmos-plugin
+
+#pragma once
+
+#include <vector>
+
+#include <glm/vec3.hpp>
+
+#include <cubos/core/gl/render_device.hpp>
+
+#include <cubos/engine/cubos.hpp>
+
+namespace cubos::engine
+{
+    class GizmosRenderer;
+
+    /// @brief Resource which queues for drawing gizmos, basic primitives useful for debugging and tools.
```

Comment:
```suggestion
    /// @brief Resource which queues commands for drawing gizmos, basic primitives useful for debugging and tools.
```
Might need to wrap the line too, otherwise the formatter will be mad